browser/js/Shopping-cart/shopping-cart.js

```diff
@@ -7,17 +7,8 @@ app.config(function ($stateProvider) {
 });
 
 app.controller('cartCtrl', function ($scope, CartFactory) {
-
   CartFactory.getCart()
   .then(function(data) {
     $scope.cart = data || null;
   });
-
-  $scope.removeFromCart = function(item) {
-    return CartFactory.removeFromCart(item)
-  };
-
-  $scope.getCartTotal = function () {
-    return CartFactory.getCartTotal();
-  }
```

Comment:
This stuff was just moved to the link function on the shopping-cart directive, correct?